scilpy/utils/streamlines.py

```diff
@@ -664,3 +665,40 @@ def cut_invalid_streamlines(sft):
                                           data_per_point=new_data_per_point)
 
     return new_sft, cutting_counter
+
+
+def upsample_tractogram(
+    sft, nb, point_wise_std=None,
+    streamline_wise_std=None, gaussian=None, spline=None, seed=None
+):
+    """ Generate new streamlines to add to a tractogram samping a gaussian
+    centered around multiple streamlines existing points.
```

Comment:
clean docstring

Reply:
Cleaned up and reworded in df4d41a44f3f2bfd5fcf3c4899142cfeccbc5675